components/FormWizard/FormWizard.jsx

```diff
@@ -7,12 +7,13 @@ import { useBeforeunload } from 'react-beforeunload';
 
 import DynamicStep from 'components/DynamicStep/DynamicStep';
 import Breadcrumbs from 'components/Breadcrumbs/Breadcrumbs';
+import { getData, saveData, deleteData } from 'utils/saveData';
 
 const FormWizard = ({ formPath, formSteps, onFormSubmit, title }) => {
   Router.events.on('routeChangeComplete', () => {
     window.scrollTo(0, 0);
   });
-  const [formData, setFormData] = useState({});
```

Comment:
I would keep the formData as state rather than move it in the localstorage. It's more efficient to keep it at component level rather than having to get it each time from the localstorage.